### PR TITLE
providing training datas

### PR DESCRIPTION
**Pull request by mat-crick**

### CODE REVIEW

File: unredactor.tsv [hunk 1]
```diff
@@ -1892,3 +1892,94 @@ josudha10	validation	Newhart	Every thing had been in the mind of an autistic chi
...
+mat-crick   training    Denton            As the sadistic gunman who enjoys kicking ██████ around
+mat-crick   training    Annemarie Schwarzenbach	I did not have a hint indication who ███████████████████████ was or what was her life story
+mat-crick   training    Jeanette Hain	all I can say is that the actress █████████████ was great with her mute facial expression
+mat-crick   training    Danielle Harris   plus a chance to check out the gorgeous ███████████████
+mat-crick   training    Peter Sellers     █████████████ plays Dick Scratcher (ha,ha), a cook for a pirate ship
+mat-crick   training    Spike Milligan    Former Goon ██████████████ replaced Medak as director
+mat-crick   training    Peter Boyle       ███████████ makes a brief appearance in the film's first 10 minutes
+mat-crick   training    Jimmy Walker      they overdubbed his scene with the Hollywood equivalent of black street lingo voiced by a ████████████
+mat-crick   training    Adam Arkin        Costars ██████████ and James Cromwell in good supporting roles
+mat-crick   Validation	Vincent Price     █████████████ must have taped his contribution in a single morning and probably didn't remember it a week later
```
Comment (cegme) [inline unredactor.tsv:1945]: Validation needs to be lowercase

**Review by cegme — CHANGES_REQUESTED** [review]
Almost there. Fix small changes. re-request a review

File: unredactor.tsv (hunk 1 above)
Comment (mat-crick) [reply unredactor.tsv]: changed into lowercase

**Review by mat-crick — COMMENTED** [review]
updated

**Comment by cegme** [comment]
@mat-crick Go ahead and re-request a review with the latest changes from other students pulled and merged

**Pull request closed without merging**